src/univariate/continuous/inversegaussian.jl

```diff
@@ -160,3 +160,41 @@ function rand(rng::AbstractRNG, d::InverseGaussian)
     u = rand(rng)
     u >= p1 ? μ^2 / x1 : x1
 end
+
+#### Fit model
+
+"""
+Sufficient statistics for `InverseGaussian`, containing the weighted
+sum of observations, the weighted sum of inverse points and sum of weights.
+"""
+struct InverseGaussianStats <: SufficientStats
+    sx::Float64      # (weighted) sum of x
+    sinvx::Float64   # (weighted) sum of 1/x
+    sw::Float64      # sum of sample weight
+
+    InverseGaussianStats(sx::Real, sinvx::Real, sw::Real) = new(sx, sinvx, sw)
+end
+
+function suffstats(::Type{InverseGaussian}, x::AbstractVector{<:Real})
+    sx = sum(x)
+    sinvx = sum(inv.(x))
```

Comment:
I think this can be replaced by `sum(inv, x)` and avoid allocating `inv.(x)`.

```julia
julia> x = randn(10_000) .+ 100;
julia> @btime sum(inv.($x))
  6.841 μs (2 allocations: 78.20 KiB)
100.02057059002993
julia> @btime sum(inv, x)
  6.146 μs (1 allocation: 16 bytes)
100.02057059002993
```